api/users.js

```diff
@@ -0,0 +1,277 @@
+const usersRouter = require("express").Router();
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
+const {
+  createUser,
+  updateUser,
+  getUserByUserId,
+  getUserByUsername,
+  getAllUsers,
+  getAllMerchandise,
+  getUserPreferencesByUserId,
+  createUserPreferences,
```

Comment:
I don't see that any of these adapters have been defined yet within db. These need to be coded out for any of this to work.